src/main/java/com/labsynch/labseer/service/LsThingServiceImpl.java

```diff
@@ -177,12 +178,12 @@ public PreferredNameResultsDTO getCodeNameFromName(String thingType, String thin
 		}
 		// Label Type and Kind
 		if (labelType != null && labelType.length() > 0){
```

Comment:
Can you make the `labelNotIgnored` predicate above on lines 166-169 also use this join / `on` syntax? 
I don't think we need `labelIgnoredNull` or `labelIgnored` if we just put `labelNotIgnored` onto the join.

Reply:
Yes, will do